index.html

```diff
@@ -114,7 +114,63 @@ <h3>This is a track title</h3>
     <br><br>
     <p class="test-text">Example text for DINPro mixin</p>
     <br>
-    <div class="circle"></div>
+
+    <div class="intro">
+      <div class="container" style="background-color: white;">
+
+          <div class="row">
+            <div class="col-sm" style="padding-right: 35px;">
+              <div class="intro-header" style="padding-bottom: 1%">
```

Comment:
to make it cleaner, can you move this "padding-bottom: 1%" to the scss file instead under .intro-header?